apps/indexer/src/api/controller/proposals.ts

```diff
@@ -0,0 +1,85 @@
+import { OpenAPIHono as Hono, createRoute } from "@hono/zod-openapi";
+
+import { ProposalsService } from "@/api/services/proposals/proposals";
+import {
+  ProposalsResponseSchema,
+  ProposalsRequestSchema,
+  ProposalRequestSchema,
```

Comment:
The service.getProposals call will fail because the ProposalsService.getProposals method expects a ProposalsRequest object with named properties, but you're passing individual parameters (skip, limit, orderDirection).